eel/tests/chain_sync_test.rs

```diff
@@ -151,10 +151,10 @@ mod chain_sync_test {
         tx_id
     }
 
-    fn open_5_chans_close_2(node_id: &str) -> Vec<String> {
+    fn open_2_chans_close_1(node_id: &str) -> Vec<String> {
         let mut open_channels = Vec::new();
 
-        for i in 0..5 {
+        for i in 0..2 {
             let tx_id =
                 nigiri::lnd_node_open_channel(NodeInstance::LspdLnd, &node_id, false).unwrap();
             if i % 2 == 0 {
```

Comment:
The new LDK version added a limit of 4 on the number of incoming unconfirmed channels you can have from the same peer.